CHANGELOG.md

```diff
@@ -35,4 +43,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased]: https://github.com/NIAEFEUP/push-to-niployments/compare/v2...HEAD
```

Comment:
Missing entry for 2.1

Reply:
I've just realized Github's editor doesn't show the links in the edit mode, so I didn't even realize there were links... Will change now.

Reply:
Wait... They're at the bottom of the file... I didn't know you could do that, my bad :clown_face:

Reply:
Yeah, I only discovered this when actively using the [Keep A Changelog](https://keepachangelog.com/en/1.1.0/) format, its pretty neat :)